fml/trace_event.h

```diff
@@ -28,6 +28,10 @@
 #define TRACE_EVENT_ASYNC_BEGIN1(a, b, c, d, e) TRACE_ASYNC_BEGIN(a, b, c, d, e)
 #define TRACE_EVENT_ASYNC_END1(a, b, c, d, e) TRACE_ASYNC_END(a, b, c, d, e)
 #define TRACE_EVENT_INSTANT0(a, b) TRACE_INSTANT(a, b, TRACE_SCOPE_THREAD)
+#define TRACE_EVENT_INSTANT1(a, b, k1, v1) \
```

Comment:
`TRACE_EVENT_INSTANT1` seems to be unused.

Reply:
I think it's nice to have 0, 1 and 2 rather than just 0 and 2. If you feel strongly about this, I can remove it.